### ISSUE TITLE
Better Documentation/Explanations on HD Operations

### ISSUE THREAD

**Issue by throughnothing** [blockchain/service-my-wallet-v3#49]
Some examples:

* It's unclear if the Receiving Address changes/updates when funds are received (i.e to automatically prevent address re-use).
* What do the responses for all the HD requests look like?

It seems that any new applications should be making use of the HD wallet + account features, as that is the future, but currently it's creating more work for developers to have to debug and reverse-engineer request/responses manually.

**Comment by jtormey** [comment]
I completely agree. HD is a great tool, but only if we can effectively communicate how to use it. Thanks for bringing this to my attention, I'll update the HD docs to include example responses and more useful descriptions.

**Comment by graymundo032** [comment]
Yo no sabia
